drivers/gpu/drm/i915/gvt/cfg_space.c

```diff
@@ -280,9 +280,20 @@ int intel_vgpu_emulate_cfg_write(struct intel_vgpu *vgpu, unsigned int offset,
 
 	/* First check if it's PCI_COMMAND */
 	if (IS_ALIGNED(offset, 2) && offset == PCI_COMMAND) {
-		if (WARN_ON(bytes > 2))
+		if (WARN_ON(bytes != 2 && bytes != 4))
 			return -EINVAL;
-		return emulate_pci_command_write(vgpu, offset, p_data, bytes);
```

Comment:
Any reason to split a 4 bytes write to 2 2-byte write?  Technically they are not equal, and the new logic may be incorrect.

Reply:
Thanks for comments. Read/write PCI cfg reg with dword size is legal. Some of the pci driver do access PCI_COMMAND  reg via pci_bus_{read,write}_config_dword(). Then for this case, it seems it is more reasonable to ignore higher 16 bits.

Reply:
@dongyaozu , the PCI_COMMAND and PCI_STATUS register are two registers. At most cases they are accessed by using pci_read/write_config_word. But as some drivers use the dword to write the two registers, I think that it is safer to split two 2-bytes write. (The pci_cmd word needs to be check whether the memory/IO trap should be enabled. The pci_status register needs to be updated with the high byte).